sendgrid/helpers/mail/email.py

```diff
@@ -35,6 +49,14 @@ def name(self):
 
     @name.setter
     def name(self, value):
+        if not (value is None or isinstance(value, str)):
+            raise TypeError('name must be of type string.')
+
+        # Escape common CSV delimiters as workaround for
+        # https://github.com/sendgrid/sendgrid-python/issues/578
+        if value is not None and (',' in value or ';' in value):
```

Comment:
Also `or '"' in value`. (`"` in the name will cause the same problems as comma.)